# Remove clang warning flag -Wno-expansion-to-defined that will throw an error in older clangs

--- a/third_party/opus/opus.gyp
+++ b/third_party/opus/opus.gyp
@@ -62,7 +62,7 @@
       'variables': {
         'clang_warning_flags': [
           # TODO(thakis): Remove once silk/macros.h has been fixed
-          '-Wno-expansion-to-defined',
+          #'-Wno-expansion-to-defined',
         ],
       },
       'include_dirs': [
